types/types.go

```diff
@@ -35,8 +35,9 @@ type Override struct {
 
 type HelmConfig struct {
 	Upgrade    bool              `yaml:"upgrade"`
-	MaxHistory int               `yaml:"maxHistory"`
-	Namespace  string            `yaml:"namespace"`
+	Debug	   bool              `yaml:"debug"`
+	LogLevel	string              `yaml:"log"`
```

Comment:
nit: Shouldn't this be a `uint`? It will default to `0` in Go.

Reply:
it is an option, i just do not set it at all if it is not set in config file